### PR TITLE
Add Example for Using Rendering Module in Documentation

### PR DESCRIPTION
**Pull request by sitong1011**
## Description

This pull request addresses https://github.com/PsiQ/qref/issues/14 by adding an example demonstrating how to use the qref.experimental.rendering module within the documentation.

Changes Made:

- Added a new section to the documentation that includes an alias sampling example of using the qref.experimental.rendering module.

## Please verify that you have completed the following steps

- [x] I have self-reviewed my code.
- [ ] I have included test cases validating introduced feature/fix.
- [x] I have updated documentation.

### CODE REVIEW

File: docs/library/userguide.md [hunk 1]
```diff
@@ -76,13 +75,36 @@ if not verification_output:
...
+if not verification_output:
+    print("Program topology is incorrect, due to the following issues:")
+    for problem in verification_output.problems:
+        print(problem)
+
+```
+
+### Rendering QREF files using `qref-render` (experimental)
+
+<div style="padding: 15px; border: 1px solid transparent; border-color: transparent; margin-bottom: 20px; border-radius: 4px; color: #8a6d3b;; background-color: #fcf8e3; border-color: #faebcc;">
```
Comment (mstechly) [inline docs/library/userguide.md:104]: [question] Why did you change it from `!!!Warning` to this elaborate html?

Comment (sitong1011) [reply docs/library/userguide.md:104]: I didn't see the alert highlight in the preview, but only the text `!!!Warning,` so I thought it might be a mistake; If it's intentional, I will leave it unchanged.

Comment (mstechly) [reply docs/library/userguide.md:104]: Did you build the docs (i.e. `mkdocs serve`) or just used preview in the editor you're using?
Cause it renders properly with the `mkdocs serve` for me.

Comment (sitong1011) [reply docs/library/userguide.md]: Solved in commit 1755e67.

**Review by mstechly — APPROVED** [review]
